sike-java/src/main/java/com/wultra/security/pqc/sike/math/optimized/fp/Fp2ElementOpti.java

```diff
@@ -83,10 +83,9 @@ public FpElement getX1() {
      */
     public Fp2Element add(Fp2Element y) {
         // y = (x0 + i*x1) + (y0 + i*y1) = x0 + y0 + i*(x1 + y1)
-        FpElement r, i;
+        FpElement r = x0.add(y.getX0());
```

Comment:
Just a comment: While this is generally a good change, I am not sure that this improves the readability of mathematics-related code.

Reply:
I would agree for a much bigger function; but I think that in this case it just adds verbosity.
@romanstrobl => what are your thoughts on that ?